### PR TITLE
[bitnami/airflow] Allow existingSecret for redis/postgresql .enabled = true

### PR DESCRIPTION
**Pull request by asaklex**
**Description of the change**

Allow  existingSecret provisioning while  redis.enabled = true  or postgresql.enabled = true.

**Benefits**

Right now if postgresql.enabled = true or redis.enabled = true , the current helm config will generate password for postgresql / redis. This new  config (redis.existingSecret / postgresql.existingSecret) allows the possibility of providing passwords while generating redis/postgresql with redis.enabled = true. 

Also whenever the helm chart is re-installed- a new password is regenerated(for redis/password) which is different from the one specified during the first run in the cluster. Since redis/postgresql uses statefulsets with PVC, passwords are saved and persisted. On upgrades / reinstall of the helm chart, passwords are regenerated but don't match the existing ones (before this fix)

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/airflow/README.md [hunk 1]
```diff
@@ -144,13 +144,15 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `postgresql.postgresqlUsername`           | Airflow Postgresql username                                                                          | `bn_airflow`                                                 |
 | `postgresql.postgresqlPassword`           | Airflow Postgresql password                                                                          | `nil`                                                        |
 | `postgresql.postgresqlDatabase`           | Airflow Postgresql database                                                                          | `bitnami_airflow`                                            |
+| `postgresql.existingSecret`         | Name of an existing secret containing the PostgreSQL password ('postgresql-password' key) . This secret is used in case of postgresql.enabled=true and we would like to specify password for newly created postgresql instance           | `nil`
```
Comment (marcosbc) [inline bitnami/airflow/README.md:147]: Could you fix the allignment of the new entry in the values table, so it looks nicer in text editors? Please also check "redis.existingSecret"

File: bitnami/airflow/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 5.0.6
+version: 5.0.7
```
Comment (marcosbc) [inline bitnami/airflow/Chart.yaml:3]: This seems to implement a new feature, so I guess a minor release would be more appropriate (5.1.0). Could you change it?

File: bitnami/airflow/README.md (hunk 1 above)
Comment (asaklex) [reply bitnami/airflow/README.md]: Thanks for the review. Alignment adjusted

File: bitnami/airflow/Chart.yaml (hunk 2 above)
Comment (asaklex) [reply bitnami/airflow/Chart.yaml]: version upgraded to 5.1.0

**Comment by marcosbc** [comment]
I had to merge the latest branch in master since it was conflicting with these changes. I'm currently working on launching our CI tests, so hopefully this can be merged soon. 🤞

**Pull request merged**